test/e2e/metrics/editor.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+import {
+	DataHelper,
+	envVariables,
+	EditorPage,
+	TestAccount,
+	getTestAccountByFeature,
+	envToFeatureKey,
+	PostsPage,
+	Metrics,
+} from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+const results: Record< string, number[] > = {};
+
+describe( DataHelper.createSuiteTitle( 'Metrics: Editor' ), function () {
```

Comment:
We keep reference those as just "metrics" but I think it might make sense to change them to "performance metrics" everywhere, since `metrics` is too generic.